pyaerocom/colocation_setup.py

```diff
@@ -423,6 +417,20 @@ def validate_basedirs(cls, v):
     keep_data: bool = True
     add_meta: dict | None = {}
 
+    model_kwargs: dict = {}
+
+    @field_validator("model_kwargs")
+    @classmethod
+    def validate_kwargs(cls, v):
+        forbidden = [
+            "vert_which",
+            "ts_type",
+        ]  # Forbidden key names which is not found in colocation_setup, or has another name there
```

Comment:
What about `start` and `stop`? I think we also need to check that no `model_kwargs` are also model_fields of `ColocationSetup`

Reply:
Implemented the changes